pkg/client/client_test.go

```diff
@@ -332,9 +339,6 @@ func TestMetadata(t *testing.T) {
 	assert.NoError(t, err)
 	assert.NotNil(t, m)
 
-	// Total count.
-	assert.Equal(t, m.TotalPages, 1)
-
 	// indexes.
 	assert.Greater(t, len(m.Indexes.Rows), 0)
 	assert.Greater(t, len(m.Indexes.Columns), 0)
```

Comment:
Use NotEmpty

Reply:
testifylint linter could help you a lot with such "issue"

It would also detect when you should use require.NoError

https://github.com/Antonboom/testifylint

It can be added easily to golangci-lint, bit you don't have golangci-lint yet,
maybe

I can help you here if you want